closure/goog/useragent/useragent.js

```diff
@@ -105,34 +94,34 @@ goog.userAgent.getNavigator = function() {
   // Need a local navigator reference instead of using the global one,
   // to avoid the rare case where they reference different objects.
   // (in a WorkerPool, for example).
-  return goog.global['navigator'] || null;
+  if ('navigator' in goog.global) {
+    return goog.global['navigator'];
+  }
+  return null;
 };
 
 
 /**
  * Whether the user agent is Opera.
  * @type {boolean}
  */
-goog.userAgent.OPERA = goog.userAgent.BROWSER_KNOWN_ ?
-    goog.userAgent.ASSUME_OPERA :
+goog.userAgent.OPERA = goog.userAgent.ASSUME_OPERA ? true :
```

Comment:
You don't want to do this. The whole reason why there's `BROWSER_KNOWN_` is that if the browser is known the call (in this case to `goog.labs.userAgent.browser.isOpera`) does not have to be made.
With this modification it would be made if the browser is known but it's not opera.